packages/strapi-plugin-content-manager/config/routes.json

```diff
@@ -8,6 +8,14 @@
         "policies": []
       }
     },
+    {
+      "method": "GET",
+      "path": "/content-types-settings",
+      "handler": "content-types.findContentTypesSettings",
+      "config": {
+        "policies": []
```

Comment:
Shouldn't we restrict this route to connected admin users?

Reply:
I just copy pasted the one above. I think it is intended so we can make the routes public in the U&P plugin